src/lib/index.js

```diff
@@ -484,6 +484,27 @@ lib.setScale = function(element, x, y) {
     return transform;
 };
 
+lib.setPointScale = function(selection, x, y) {
```

Comment:
I'm ok with this being in `lib/`. Putting this in `src/traces/scatter/set_point_scale.js` would be ok too. Your call @rreusser